sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -258,6 +258,10 @@ abstract class JdbcDialect extends Serializable {
     s"ALTER TABLE $tableName ALTER COLUMN ${quoteIdentifier(columnName)} SET $nullable"
   }
 
+  def getTableCommentQuery(table: String, comment: String): String = {
+    s"COMMENT ON TABLE $table IS '$comment'"
```

Comment:
it's a separate statement. Is it possible to specify table comment during table creation?

Reply:
Seems only MySQL has a way to let user to specify table options such as `COMMENT`, `ENGINE`, etc. in `CREATE TABLE`. I use a separate statement for MySQL anyway to make it consistent with others.